### DISCUSSION TITLE
Unable to preview the email content of a ticket

### DISCUSSION THREAD

**Discussion by t9NVYj68** [YetiForceCompany/YetiForceCRM#15083] [How to]
Hello,

When I try to send an email to create a new ticket, the content is not displayed but there is an attachment as an HTML file

<img width="1752" alt="Capture d’écran 2021-08-17 à 11 58 05" src="https://user-images.githubusercontent.com/34187647/129705854-b4eeae8c-9c9a-4dce-9930-13fdfda7ba3e.png">

Do you know why?

Thanks

**Comment by mariuszkrzaczkowski** [comment]
This problem requires access to the mailbox and a deeper analysis, I suppose the mail may be of a non-standard structure.
